.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish image to GitHub Container Registry
+
+on:
+  workflow_run:
+    workflows: [Run E2E Tests]
+    types:
+      - completed
+
+jobs:
+  on-success:
+    runs-on: ubuntu-22.04
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Docker Metadata action
+        id: meta
+        uses: docker/metadata-action@v5
+        env:
+          DOCKER_METADATA_PR_HEAD_SHA: "true"
+        with:
+          images: |
+            ghcr.io/buildwithgrove/path
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=ref,event=tag,suffix=-rc
+            type=sha,suffix=-rc
+            type=sha,format=long,suffix=-rc
+
+      ## TODO: Is the GH token in place in the repo?
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+
+      ## TODO: Do we want/need to make PATH multi arch?
```

Comment:
Yes please, can you do it in this PR?

PATH is something I'd like to be able to run on my M1 processor.

Hit a lot of issues with in pocketdex so let's just get ahead of it: https://github.com/pokt-network/pocketdex/blob/1178ca47f68f6596bed7658dc1ba2106cdda1362/.github/workflows/ci-docker.yml#L33

cc @bryanchriswhite @okdas for visibility

Reply:
Yup. Also, more providers (including the lean services such as hetzner) add arm64 to their offerings, so I say we need it.

Reply:
Added ARM arch for docker build.